plugins/shared-react/src/utils/pipeline-utils/taskRun-utils.ts

```diff
@@ -4,7 +4,7 @@ export const getTaskRunsForPipelineRun = (
   pipelinerun: PipelineRunKind | null,
   taskRuns: TaskRunKind[],
 ): TaskRunKind[] => {
-  const associatedTaskRuns = taskRuns.reduce(
+  const associatedTaskRuns = taskRuns?.reduce(
```

Comment:
if `taskRuns` are undefined,  which ideally shouldn't be then `getTaskRunsForPipelineRun` will return undefined but the function return type is an array so ideally caller needs to ensure that taskRuns are there, and to be safe we can have the default value of `taskRuns` as []